grakn-dist/src/bin/grakn-engine.sh

```diff
@@ -38,6 +38,9 @@ for jar in "${GRAKN_HOME}"/lib/*.jar; do
     fi
 done
 
+# Add path containing logback.xml
+CLASSPATH="$CLASSPATH":"${GRAKN_HOME}"/conf/main/
```

Comment:
I agree, but for the future! Ideally all the hard-coded paths in the shell scripts should be defined just once somewhere.